apps/common-app/src/examples/RuntimeTests/RuntimeTestsExample.tsx

```diff
@@ -1,30 +1,55 @@
 import React from 'react';
 import RuntimeTestsRunner from './ReanimatedRuntimeTestsRunner/RuntimeTestsRunner';
 
-// load tests
-import './tests/TestsOfTestingFramework.test';
-
-import './tests/animations';
-
-import './tests/core/cancelAnimation.test';
-
-import './tests/utilities/relativeCoords.test';
-
-import './tests/layoutAnimations/entering/enteringColors.test';
-import './tests/layoutAnimations/entering/predefinedEntering.test';
-
-import './tests/layoutAnimations/exiting/predefinedExiting.test';
-
-import './tests/layoutAnimations/layout/predefinedLayoutPosition.test';
-
-import './tests/advancedAPI/useFrameCallback.test';
-// import './tests/advancedAPI/measure.test'; crash on Android
-
-import './tests/core/useSharedValue.test';
-import './tests/core/useAnimatedStyle/reuseAnimatedStyle.test';
-import './tests/core/useDerivedValue/basic.test';
-import './tests/core/useDerivedValue/chain.test';
-
 export default function RuntimeTestsExample() {
-  return <RuntimeTestsRunner />;
+  return (
+    <RuntimeTestsRunner
+      importButtons={[
+        {
+          testSuiteName: 'Tests of testing framework',
+          importTest: () => {
+            require('./tests/TestsOfTestingFramework.test');
+          },
+        },
+        {
+          testSuiteName: 'animations',
+          importTest: () => {
+            require('./tests/animations');
```

Comment:
I'm considering if it would be a better idea to move `requires` from `./tests/animations/index.ts` here to keep everything in one place.